pkgs/top-level/python-packages.nix

```diff
@@ -7240,6 +7240,8 @@ in {
 
   pandoc-attributes = callPackage ../development/python-modules/pandoc-attributes { };
 
+  pandoc-plantuml-filter = callPackage ../development/python-modules/pandoc-plantuml-filter { };
```

Comment:
move this out of `python-modules` and into the related pkgs dir
```suggestion
  pandoc-plantuml-filter = python3Packages.callPackage ../development/python-modules/pandoc-plantuml-filter { };
```

Reply:
Do we keep the alias here or do we change the alias to a top level package?